### PR TITLE
Document `switch.pi_hole_FRIENDLYNAME`

### LINKED ISSUE #31368: pi-hole integration doesn't document the switch entity it provides
### Feedback

I was writing an automation to toggle the Pi-holes on my network, and could only find the `pi_hole.disable` service in the documentation. Someone pointed out on https://github.com/home-assistant/core/issues/105995 that there is also a `switch.pi_hole_FRIENDLYNAME` provided by the integration that isn't in the documentation.

### URL

https://www.home-assistant.io/integrations/pi_hole

### Version

n/a docfix

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by unixorn**
## Proposed change

Add `switch.pi_hole_FRIENDLYNAME` re https://github.com/home-assistant/core/issues/105995

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- This PR fixes https://github.com/home-assistant/home-assistant.io/issues/31368

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/pi_hole.markdown [hunk 1]
```diff
@@ -63,3 +63,7 @@ data:
 target:
   entity_id: all
 ```
+
+## Switches
+
+The integratation will create a switch for the pi-hole named `switch.pi_hole_FRIENDLYNAME` which will allow you to toggle the pi-hole on and off.
```
Comment (frenck) [inline source/_integrations/pi_hole.markdown]: Entity IDs are not guaranteed and can be renamed. Just describe the switch and leave the entity ID out.

**Review by frenck — DISMISSED** [review]

File: source/_integrations/pi_hole.markdown (hunk 1 above)
Comment (unixorn) [reply source/_integrations/pi_hole.markdown]: will do

Comment (unixorn) [reply source/_integrations/pi_hole.markdown]: Done

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @unixorn :+1:

File: source/_integrations/pi_hole.markdown [hunk 2]
```diff
@@ -63,3 +63,7 @@ data:
 target:
   entity_id: all
 ```
+
+## Switches
+
+The integratation will create a switch for the pi-hole named `switch.pi_hole` which will allow you to toggle the pi-hole on and off.
```
Comment (frenck) [inline source/_integrations/pi_hole.markdown:69]: Wait, my review comment was: Remove the entity ID completely?

But it is still in?

../Frenck

Comment (c0ffeeca7) [reply source/_integrations/pi_hole.markdown:69]: thank you for the catch, @frenck 
#31424

**Pull request merged**